modules/boost.config/1.88.0/MODULE.bazel

```diff
@@ -0,0 +1,9 @@
+module(
+    name = "boost.config",
+    version = "1.88.0",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 108800,
+)
+
+bazel_dep(name = "boost.pin_version", version = "1.88.0")
```

Comment:
This is essentially pinning the lowest version? Theoretically, you could still have other boost modules with higher version?